go/mysql/streaming_query.go

```diff
@@ -133,14 +133,15 @@ func (c *Conn) FetchNext() ([]sqltypes.Value, error) {
 	}
 
 	// Regular row.
-	return c.parseRow(data, c.fields, readLenEncStringAsBytes)
+	return c.parseRow(data, c.fields, readLenEncStringAsBytes, in)
 }
 
 // CloseResult can be used to terminate a streaming query
 // early. It just drains the remaining values.
 func (c *Conn) CloseResult() {
+	row := make([]sqltypes.Value, 0, len(c.fields))
 	for c.fields != nil {
-		rows, err := c.FetchNext()
+		rows, err := c.FetchNext(row[:0])
```

Comment:
help me understand - why do we need to do this slice subset of a just created slice?

Reply:
We actually do not; I had this in place from an older iteration of the interface. Right now it doesn't make an actual difference, as the changes to the slice's contents inside of `FetchNext` are not reflected on the original slice, so this is redundant.